### PR TITLE
scaled sentiment score

### PR DESCRIPTION
**Pull request by SrishtiR**
Updated sentiment analyzer to return weighted score instead of average

### CODE REVIEW

File: sentiment_analyzer.py [hunk 1]
```diff
@@ -44,7 +44,12 @@ def compute_sentiment(self, country, city, hashtag):
             for tweet in tweets:
                 compound_score = self.sentiment_analyzer(tweet)
                 compound_sum += compound_score
-            return compound_sum / num_tweets
+            avg_score = compound_sum / num_tweets
+            if avg_score <= 0.5 and avg_score >= -0.5:
```
Comment (AbigailFernandes) [inline sentiment_analyzer.py]: Change this to 0.25 instead of 0.5, and -0.25 for -0.5

**Review by AbigailFernandes — CHANGES_REQUESTED** [review]

File: sentiment_analyzer.py (hunk 1 above)
Comment (SrishtiR) [reply sentiment_analyzer.py]: Updated and added the weight to config

**Comment by shreshtha-pankaj** [comment]
You made changes on the most updated conf file right?

**Comment by shreshtha-pankaj** [comment]
sent you the updated file over slack, please verify

**Comment by SrishtiR** [comment]
Verified, I updated the most recent one

**Review by shreshtha-pankaj — APPROVED** [review]

**Pull request merged**